package.json

```diff
@@ -59,6 +62,40 @@
       "import/extensions": [
         "error",
         "ignorePackages"
+      ],
+      "jsdoc/tag-lines": [
+        "error",
+        "any",
+        {
+          "startLines": 1
+        }
+      ],
+      "jsdoc/require-returns": [
+        "error",
+        {
+          "enableFixer": true
+        }
+      ],
+      "jsdoc/require-param": [
+        "error",
+        {
+          "enableFixer": true
+        }
+      ],
+      "jsdoc/require-jsdoc": [
+        "warn",
```

Comment:
this will eventually become an error, but in the meantime I'm adding it as a warning since it will be added to the PRs as call outs.

Reply:
nice!!